scripts/splitSources.py

```diff
@@ -27,8 +27,8 @@ def extractSourceName(line):
 # writes the following source into a file named sourceName
 def writeSourceToFile(lines):
     filePath, srcName = extractSourceName(lines[0])
-    # print "sourceName is", srcName
-    # print "filePath is", filePath
+    # print("sourceName is ", srcName)
+    # print("filePath is", filePath)
```

Comment:
Wouldn't this then also need an additional whitespace?

Reply:
Ah, wait. Could these comments be removed at all?

Reply:
actually, no.